tasks/task02/Area_of_a_triangle.py

```diff
@@ -0,0 +1,42 @@
+# main menu
+import math
+print("Welcome to the triangle area calculation tool.")
+while True:
+    print("\n","Menu:","1. Calculate triangle area by base and height","2. Calculate triangle area by 2 sides and angle between them","3. Exit", sep = "\n")
+    print("Enter menu item number: ", end = "")
+    mitem = input()
```

Comment:
`input` has `prompt` parameter so that you can write:

`mitem = input("Enter menu item number:")`

Reply:
https://www.w3schools.com/python/ref_func_input.asp